product_supplierinfo_discount/README.rst

```diff
@@ -47,6 +47,7 @@ Contributors
 ------------
 
```

Comment:
"This module requires *purchase_discount* module, also available in
 this repository."